.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+## Summary
+
+> What is the reason for this Pull-Request? Are you bumping the version?
```

Comment:
```suggestion
> What is the reason for this Pull-Request?
```

Reply:
I don't think this suggestion got incorporated in the latest version? 
This is most important part of the PR template IMO, it saves immense time for reviewers to see code snippet and outputs at the top of the PR before digging into details.

Reply:
Interesting! 

I am pretty sure I committed the suggestion but somehow it is gone. 

I am recommitting it now.

Reply:
✅ Updated the PR template with the above suggestion.